index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    
+    <div style="width: 50vw; margin: 0 auto;">
+      
+      <h1 id="title" style="text-align: center; margin: auto;">Title</h1>
```

Comment:
please place the style of the elements in the CSS file and use class to identify the styles of the elements in the HTML